docs/administration-guide/08-single-sign-on.md

```diff
@@ -0,0 +1,25 @@
+## Single Sign On with Google
+
+Enabling single sign on lets your team login with a click instead of using email and password and can optionally let them sign up for Metabase accounts without an admin having to create them first.
+
+Currently Metabase works with Google accounts for single sign on. As time goes on we may add other auth providers. If you have a service you’d like to see work with Metabase please let us know by [filing an issue](http://github.com/metabase/metabase/issues/new).
+
+### Enabling Sign in
+
+To let your team start signing in with Google you’ll first need to create an application through Google’s [developer console](https://console.developers.google.com/projectselector/apis/library).
+
+To create a new project follow the instructions from google here. https://developers.google.com/identity/sign-in/web/devconsole-project]
```

Comment:
Capitalize "Google"

Is it a new project or a new application? It's referred to as a new application in the previous sentence.